doc/source/data/getting-started.rst

```diff
@@ -152,7 +152,8 @@ since your map function can return an output batch of any size.
     # -> [{'value': 0}, {'value': 2}, ...]
 
 By default, transformations are executed using Ray tasks.
-For transformations that require setup, specify ``compute="actors"`` and Ray will use an autoscaling actor pool to execute your transforms instead.
+For transformations that require setup, specify ``compute="actors[n]"`` and Ray will use an actor pool of size `n` to execute your transforms instead.
+For an autoscaling actor pool, specify ``compute="actors"``.
```

Comment:
We have users looking for the ability to set min and max for an autoscaling actor pool instead of a fixed size. What do you think about changing this to support min and max pool size instead?